server/rooms.ts

```diff
@@ -1368,18 +1401,13 @@ export class GlobalRoomState {
 				title: room.title,
 				desc: room.settings.desc || '',
 				userCount: room.userCount,
+				section: room.settings.section || undefined,
```

Comment:
```suggestion
				section: room.settings.section ? RoomSections.sectionNames[room.settings.section] : undefined,
```

Reply:
This still has a chance to return undefined if somehow a room's section gets mutated; wouldnt it make more sense to pass along `RoomSections.sectionName[room.settings.section] || room.settings.section` (room.settings.section is typed `RoomSection | undefined`)?

Reply:
Yeah, that's fine.